src/core/modules/StarMgr.cpp

```diff
@@ -433,6 +446,9 @@ void StarMgr::init()
 	setFlagLabels(conf->value("astro/flag_star_name",true).toBool());
 	setFlagAdditionalNames(conf->value("astro/flag_star_additional_names",true).toBool());
 	setDesignationUsage(conf->value("astro/flag_star_designation_usage", false).toBool());
+	setFlagDblStarsDesignation(conf->value("astro/flag_star_designation_dbl_stars", false).toBool());
```

Comment:
I think we can shorten the names to flag_star_designation_dbl

Reply:
Good idea, thanks!